static-assets/components/cstudio-contextual-nav/quick-create.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2007-2019 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+var YDom = YAHOO.util.Dom;
+var YEvent = YAHOO.util.Event;
+
+/**
+ * Branded Logo Plugin
+ */
+CStudioAuthoring.ContextualNav.WcmQuickCreate = CStudioAuthoring.ContextualNav.WcmQuickCreate || {
+
+    initialized: false,
+
+    /**
+     * initialize module
+     */
+    initialize: function(config) {
+        YAHOO.util.Event.onAvailable("quick-create", function() {
+            CStudioAuthoring.Service.getQuickCreate( {
```

Comment:
1. Will it really ever not be available?
2. Don't wait for the element to be available to retrieve the data. Retrieve the data and initialize the element when it is available.

Reply:
Done